cmd/snap/main.go

```diff
@@ -165,7 +165,7 @@ func Parser() *flags.Parser {
 		printVersions()
 		panic(&exitStatus{0})
 	}
-	parser := flags.NewParser(&optionsData, flags.HelpFlag|flags.PassDoubleDash|flags.PassAfterNonOption)
+	parser := flags.NewParser(&optionsData, flags.PassDoubleDash|flags.PassAfterNonOption)
```

Comment:
Just in case: today when we type "snap help" and "snap --help" we get a nice long description of commands. That's nice and we should make sure that's preserved with the change, instead of getting, say, the pretty boring text of the help command itself.

Reply:
yes, that's preserved.